public/index.html

```diff
@@ -14,6 +14,7 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,600,700|Open+Sans&display=swap" rel="stylesheet"> 
```

Comment:
I think you can move this to the fonts less file - I think this way won't work if we're doing a build version of our project because this index.html is different than the build folder index.html